tests/App/tailwind.config.js

```diff
@@ -0,0 +1,43 @@
+const colors = require('tailwindcss/colors')
+
+module.exports = {
+    content: [
+        './assets/**/*.js',
+        './templates/**/*.{html,html.twig}',
+        './vendor/whatwedo/**/*.{html,html.twig,js,scss}',
+        './var/cache/twig/**/*.php',
+        './src/Definition/*.php',
+    ],
+    safelist: [
+        {
+            pattern: /grid-(cols|rows)-\d/,
+            variants: ['lg', 'md', 'sm'],
+        },
+    ],
+    theme: {
+        extend: {
+            colors: {
+                // needs fixing 
```

Comment:
@marcwieland95 do you know why this is needed here? Running yarn dev in the tests/App directory will fail without this ...